### PR TITLE
Updating docs to include interactive demos

### PR DESCRIPTION
**Pull request by lappemic**
#### Changes
- updated `docs/source/conf.py` to include gradios js file

Fixes ([private Slack message](https://kornia.slack.com/archives/C078L00V7NY/p1721066635279829))

#### Type of change
- [x] 📚  Documentation Update

#### Persisting Issue
- Demo input images don't show up correctly
- Can't select options from dropdowns
- Buttons don't work when clicked

Working with gradio on it (Issue: https://github.com/gradio-app/gradio/issues/8835). This PR is opened for transparency and solving the persisiting issues.

cc @edgarriba, @johnnv1

### CODE REVIEW

**Comment by lappemic** [comment]
I used an old gradio js client and had the `type` definition not included in the `Sphinx` conf file. Also i had to update the `<gradio-app>` attributes from `space` to `src` to be compatible.

With these changes the demos should work again.

**Review by johnnv1 — APPROVED** [review]
Amazing patch! Thank you for fixing these embedded demos, and for ensuring demos are working on the spaces

--- 
We can check it working for this PR in the readthedocs: https://kornia--2958.org.readthedocs.build/en/2958/filters.html#id5

File: kornia/geometry/transform/affwarp.py [hunk 1]
```diff
@@ -670,7 +670,7 @@ class Resize(nn.Module):
 
     .. raw:: html
 
-        <gradio-app space="kornia/kornia-resize-antialias"></gradio-app>
+        <gradio-app src="kornia/kornia-resize-antialias"></gradio-app>
```
Comment (johnnv1) [inline kornia/geometry/transform/affwarp.py:673]: just as a note, we can figure out in a further PR: on kornia HF we have 17 spaces, and I think not all of them are in the docs

**Comment by lappemic** [comment]
@johnnv1 do you think we should update somethingo on this PR or are we ready to merge?

Also, can we do something about the not passing test? I feel they are around for quite some time...

**Comment by johnnv1** [comment]
> 

yeap, this patch is fine to go, the CI is falling unrelated with this patch

**Comment by lappemic** [comment]
I do not really understant how we proceed with PR which are approved but have a failing CI not related with the PR. Could you expand a bit on this? 😅

**Comment by johnnv1** [comment]
> I do not really understant how we proceed with PR which are approved but have a failing CI not related with the PR. Could you expand a bit on this? 😅

There is nothing for our side, I ping the guys in Slack -- they can force the merge without passing the policies 😄

**Pull request merged**